WordPress/gutenberg
# docgen: Add function to handle list of types for theme.json schema

## What?

This PR enables docgen to list multiple types for the theme.json style properties listed in theme-json-living.md.

## Why?

Currently, the generated theme.json docs will not list multiple types and instead will print `undefined`. See the [current generated entry](https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/theme-json-reference/theme-json-living.md#border-1) for the `radius` style property.

## How?

Adds a new function called `formatType`, which takes the style property object and checks if the `type` key exists. If it does, it will default to that (the same as the current behavior). If it doesn't exist, it checks for the `anyOf` or `oneOf` properties and makes a comma-separated list of those types.

## Testing Instructions




Run `node ./bin/api-docs/gen-theme-reference.js`, and the generated type for the `radius` property should be "string, object" instead of "undefined".

### Testing Instructions for Keyboard


## Screenshots or screencast
